### ISSUE TITLE
[MU4 Issue] Metronome bell is played instead of click and vice-versa in imported MS3 score

### ISSUE THREAD

**Issue by rgreen5** [musescore/MuseScore#12871]
**Describe the bug**

The attached score, written in MS3, is a series of metronome patterns, with the bell sounding on the main beat and click on subsequent beats. However when this file is played in MS4, the click is where the bell should be and vice versa.

**To Reproduce**

* Open the attached file in both MS3 and MS4 and compare the playback. 
[metronome.zip](https://github.com/musescore/MuseScore/files/9338870/metronome.zip)

RESULT: Bell and click are inverted in MS4 playback.

**Expected behavior**
Playback should be the same in both cases.

**Platform information**
OS: Linux Mint 20.1, Arch.: x86_64, MuseScore version (64-bit): 4.0.0-2858462377, revision: github-musescore-musescore-93c9b73

**Comment by zacjansheski** [comment]
I know it's been almost a year since this has been opened, but could you provide more information about the file?  How did you set up the instrument in MS3?  Did you use a special soundfont like MDL or a special synthesizer setting?

**Comment by rgreen5** [comment]
The MS3 file was created using the native MuseScore_General.sf3 soundfont. I've checked the soundfont in  Polyphone and the metronome bell and click are correctly positioned (A0/33 for the Met. click; and A#0/34 for the bell).

I am playing back in MS4 using the MS Basic soundfont. Again I have checked this soundfont in Polyphone and everthing is in order.

So the problem _seems_ to be in the allocation of sounds by the "Edit drumset" dialog in MS4. The notehead shapes are correct (Bell is standard shape; click is crosshead shaped) but the sounds are being applied the wrong way around.
